tests/quasi-newton/helpers.cpp

```diff
@@ -8,17 +8,21 @@
 /// tests for different QN settings if correct fixed point is reached
 void runTestQN(std::string const &config, TestContext const &context)
 {
-  std::string meshName, writeDataName, readDataName;
+  std::string meshName, writeDataName1, writeDataName2, readDataName1, readDataName2;
 
   if (context.isNamed("SolverOne")) {
-    meshName      = "MeshOne";
-    writeDataName = "Data1";
-    readDataName  = "Data2";
+    meshName       = "MeshOne";
+    writeDataName1 = "Data11";
+    writeDataName2 = "Data12";
+    readDataName1  = "Data21";
+    readDataName2  = "Data22";
```

Comment:
```suggestion
    writePrimaryDataName = "PrimaryData1";
    writeSecondaryDataName = "SecondaryData1";
    readPrimaryDataName  = "PrimaryData2";
    readSecondaryName = "SecondaryData2";
```
I think such renaming could makes things easier to grasp.

Reply:
In most of the tests we use serial coupling, so the data written by `SolverOne` are neither primary nor secondary. Should we still use this renaming or just describe them in the comment?